frontend/app/views/spree/checkout/edit.html.erb

```diff
@@ -1,3 +1,6 @@
+<% content_for :google_analytics do %>
+  <%= render :partial => 'spree/shared/google_checkout.js', :locals => { :order => @order, :step_number => checkout_states.index(@order.state) + 1 } %>
```

Comment:
why not `<%= render 'spree/shared/google_checkout.js', order: @order, step_number: checkout_states.index(@order.state) + 1 %>` :)